tests/unit/Makefile.am

```diff
@@ -7,8 +7,9 @@ check_SCRIPTS += \
 	test_package_module_yum
 endif
 
-if HAVE_PKG_INSTALL
 if HAVE_SHUNIT2
+check_SCRIPTS += test_package_module_freebsd_ports
+if HAVE_PKG_INSTALL
```

Comment:
Shouldn't this contain a check to make sure ports is available? Or is it enough to check for `shunit2`?

See also https://github.com/cfengine/masterfiles/blob/master/configure.ac#L122-L126 for how this is checked for `pkg_install`.